includes/widgets/class-wc-widget-rating-filter.php

```diff
@@ -109,7 +172,7 @@ public function widget( $args, $instance ) {
 
 			echo '<span class="star-rating" title="' . esc_attr( sprintf( __( 'Rated %s and above', 'woocommerce' ), $rating ) ). '">
 					<span style="width:' . esc_attr( ( $rating / 5 ) * 100 ) . '%">' . sprintf( __( 'Rated %s and above', 'woocommerce'), $rating ) . '</span>
-				</span>';
+				</span> (' . $count . ')';
```

Comment:
I know $count is already cast as an int above -- but it might be a good idea to escape on output here as well, just incase anything ever changes in the lines above (or a filter is added).